### PR TITLE
Enable setting the verbose and silent options via config file

### LINKED ISSUE #127: If using a local config, the output should include this information
During testing, it wasn't clear to me if local-tableland was actually picking up my local config file or not.

### PR DESCRIPTION
**Pull request by joewagner**
The default values that yargs was setting were not allowing the config file to specify the verbose and silent options.
Fixes #127

### CODE REVIEW

**Comment by joewagner** [comment]
@carsonfarmer Is this what you were thinking to solve #127

File: src/up.ts [hunk 1]
```diff
@@ -39,25 +36,22 @@ const go = async function () {
...
-    // will help the user setup their project directory then exit when finished
+    // If init arg is given we want to open a terminal prompt that will
+    // help the user setup their project directory then exit when finished
     await projectBuilder();
     return;
   }
 
+  const opts: Config = {};
+
   // @ts-ignore
```
Comment (carsonfarmer) [inline src/up.ts:47]: why the ignores here?

Comment (carsonfarmer) [reply src/up.ts:47]: it is safer to use `as any` in these cases, because you might be ignoring something else entirely

File: src/validators.ts [hunk 2]
```diff
@@ -101,6 +101,7 @@ class ValidatorDev {
 
     // TODO: this could be parsed out of the deploy process, but since
     //       it's always the same address just hardcoding it here
+    // TODO: maybe we can get this from evm-tableland?
```
Comment (carsonfarmer) [inline src/validators.ts:104]: 👍

**Review by carsonfarmer — CHANGES_REQUESTED** [review]
Just a minor nit about not using @ts-ignore

**Comment by carsonfarmer** [comment]
> @carsonfarmer Is this what you were thinking to solve #127

Yes, exactly, thanks!

File: src/up.ts (hunk 1 above)
Comment (joewagner) [reply src/up.ts]: Nice! 
I just pushed a change.

**Review by carsonfarmer — APPROVED** [review]

**Pull request merged**